backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/SqlDatabase.java

```diff
@@ -616,6 +616,24 @@ public DSLContext getJooq() {
     return jooq;
   }
 
+  void getJooqAsAdmin(JooqTransaction transaction) {
+    if (inTx()) {
```

Comment:
why the if , why if in transaction change user and otherwise create the connectionProvider , i dont get it, maybe explain in PR or even add code comment

Reply:
I will explain, but motivation is that a test failed rightfully because inside of the tx() some permission where changed why I initially asked the roles outside the tx (so then those changes are invisible).

Reply:
done